### PR TITLE
Add bug report template

### PR DESCRIPTION
**Pull request by ann0see**
Adding a bug template also makes it possible to automatically set the bug label.

### CODE REVIEW

**Comment by ann0see** [comment]
It happened as I thought. aefb1b42589cdbd5edf5e8a1e4a9aae7b72d1d22

Wanted to propose a feature request template and I committed to the master branch. Sorry (already reverted). Nevertheless: Can somebody have a look at this commit too and give feedback?

**Comment by corrados** [comment]
I guess this will be the first time you (the main developer team) have to deal with the decision of the merge on your own. I already removed me from the jamulussoftware project, so, I actually do not have the rights anymore to perform the merge.

**Comment by hoffie** [comment]
> Can somebody have a look at this commit too and give feedback?

Not sure if you are seeking feedback from other maintainers only. I'm taking the freedom to add a note:

In my Github experience, people sometimes forget to remove the boilerplate text. My suggestion would therefore be to place the instructions in a comment so that they can still be seen when opening the issue but will not be part of the issue in the end.

Example: https://raw.githubusercontent.com/prometheus/prometheus/master/.github/ISSUE_TEMPLATE/bug_report.md

Having a basic structure sounds good. I also like the proposed headers.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (hoffie) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]:
```suggestion
<!-- Add any other context about the problem here. -->
```

**Comment by pljones** [comment]
I'd agree with @hoffie's suggestions -- make it clear the headings aren't headings but information to be substituted.

**Comment by hoffie** [comment]
> I'd agree with @hoffie's suggestions -- make it clear the headings aren't headings but information to be substituted.

Actually, I would keep the headings but only comment out the descriptions.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (ann0see) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: Are you sure about this wording? It might sound a bit harsh with so many question marks.

Comment (hoffie) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: > Are you sure about this wording? It might sound a bit harsh with so many question marks.

I certainly did not mean to sound harsh and personally do not consider the question marks to be harsh. I fully agree with your point though as everyone will read things differently. Therefore, here comes a suggestion with less question marks and more focus on examples. :)

```suggestion
<!-- Add the operating system you are running, e.g. Windows 10 Version 20H2, macOS 10.15.5, Ubuntu Linux 20.10. Also add where you got Jamulus from, e.g. from jamulus.io, from your Linux distribution or if you built from source. -->
```

Comment (ann0see) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: Yes. This sounds better ;-). I think we don't need to say from jamulus.io maybe we could say "downloaded it from the internet". In fact, at the moment, there's no Jamulus binary on jamulus.io

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (ann0see) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: I don't really see any changes here?

Comment (hoffie) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: > I don't really see any changes here?

Meh, Github markup failure on my side. ;)

```suggestion
<!-- Get this from the Help > About menu or from the `Jamulus --version` command line -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (hoffie) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]:
```suggestion
<!-- Add the operating system you are running, e.g. Windows 10 Version 20H2, macOS 10.15.5, Ubuntu Linux 20.10. Also add where you got Jamulus from, e.g. if you downloaded it from the internet, if you got it from your Linux distribution or if you built from source. -->
```

**Comment by ann0see** [comment]
I think this looks good now. Thanks @hoffie!

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
@gilgongo can you approve and merge this if everything is ok?

**Comment by gilgongo** [comment]
OK sure.

**Pull request merged**
